entrypoint.sh

```diff
@@ -40,20 +42,15 @@ DATADIR="/home/erigon-gnosis/.local/share"
 exec erigon --datadir=${DATADIR} \
     --chain=gnosis \
     --http.addr=0.0.0.0 \
-    --http.port=8545 \
     --http.vhosts=* \
     --http.corsdomain=* \
     --ws \
     --private.api.addr=0.0.0.0:9090 \
     --metrics \
     --metrics.addr=0.0.0.0 \
     --metrics.port=6060 \
-    --pprof \
-    --pprof.addr=0.0.0.0 \
-    --pprof.port=6061 \
```

Comment:
Would also be nice to have this as a configurable environment variable, but definitely nothing critical.

Reply:
I guess it can be done via `EXTRA_OPTS`
https://github.com/dappnode/DAppNodePackage-Gnosis-Chain-Erigon/blob/main/docker-compose.yml#L18

Reply:
Returned those flags and made port conifgurable variable